lib/core/macros.nim

```diff
@@ -132,10 +129,6 @@ const
                    nnkCallStrLit}
   nnkPragmaCallKinds = {nnkExprColonExpr, nnkCall, nnkCallStrLit}
 
-proc `!`*(s: string): NimIdent {.magic: "StrToIdent", noSideEffect, deprecated.}
```

Comment:
I don't really know, but maybe it's too early to fully remove this deprecated procedure? @Araq will decide :)

Reply:
This should be alive for a little longer. Sorry.